src/core/server/stacks/createComment.ts

```diff
@@ -269,6 +274,17 @@ export default async function create(
   const ancestorIDs: string[] = [];
   let parent = await retrieveParent(mongo, tenant.id, input);
   if (parent) {
+    // Check to see that the most recent revision matches the one we just replied
+    // to.
+    if (input.parentRevisionID) {
+      const revision = getLatestRevision(parent);
+      if (revision.id !== input.parentRevisionID) {
+        throw new CommentRevisionNotFoundError(
+          parent.id,
+          input.parentRevisionID
+        );
+      }
+    }
```

Comment:
It is possible for a comment to come in from a stale page (or conversation view) that has old revision ID's. If we do this, they will likely get errors.

I just had a person making a long reply to one of my comments on HN and I edited it while they were writing their reply.

In fast paced comment sections, this happens a bit more frequently as well, where people are editing their comment simultaneously while others reply to them.

I'm not sure if we want to change this?